hack/install-wrapper.sh

```diff
@@ -158,10 +153,7 @@ function update_antrea_url() {
     ANTREA_BRANCH="release-$(echo $ANTREA_VERSION | cut -b 2-4)"
     ANTREA_INSTALL_SCRIPT="https://raw.githubusercontent.com/antrea-io/antrea/${ANTREA_BRANCH}/hack/externalnode/install-vm.sh"
```

Comment:
This needs to be changed. With 1.9 we can download the install script from the release assets.